instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/GarbageCollector.java

```diff
@@ -66,43 +73,52 @@ static void registerObservers(
       Function<Notification, GarbageCollectionNotificationInfo> notificationInfoExtractor) {
     Meter meter = RuntimeMetricsUtil.getMeter(openTelemetry);
 
-    LongHistogram gcTime =
+    DoubleHistogramBuilder gcDurationBuilder =
         meter
             .histogramBuilder("process.runtime.jvm.gc.duration")
             .setDescription("Duration of JVM garbage collection actions")
-            .setUnit("ms")
-            .ofLongs()
-            .build();
+            .setUnit("s");
+    setGcDurationBuckets(gcDurationBuilder);
+    DoubleHistogram gcDuration = gcDurationBuilder.build();
 
     for (GarbageCollectorMXBean gcBean : gcBeans) {
       if (!(gcBean instanceof NotificationEmitter)) {
         continue;
       }
       NotificationEmitter notificationEmitter = (NotificationEmitter) gcBean;
       notificationEmitter.addNotificationListener(
-          new GcNotificationListener(gcTime, notificationInfoExtractor), GC_FILTER, null);
+          new GcNotificationListener(gcDuration, notificationInfoExtractor), GC_FILTER, null);
+    }
+  }
+
+  private static void setGcDurationBuckets(DoubleHistogramBuilder builder) {
+    if (!(builder instanceof ExtendedDoubleHistogramBuilder)) {
+      // that shouldn't really happen
+      return;
     }
+    ((ExtendedDoubleHistogramBuilder) builder)
+        .setAdvice(advice -> advice.setExplicitBucketBoundaries(emptyList()));
```

Comment:
Hi @mateuszrzeszutek, sorry for tagging in an old PR, but I am just wondering the reason for making explicitBucketBoundaries as an empty list? 
Before version 1.26.0 I am pretty sure it used to have the explicitBucketBoundaries.

Reply:
Hey @kes2464 ,
We've had many discussions on what the buckets for the gc metrics should be; and for a long while we were of the opinion that we should not provide any, and instead allow the users to set the buckets they want via [the SDK Views](https://opentelemetry.io/docs/instrumentation/java/manual/#views). The current implementation reflects that.
We'll likely add some buckets when the JVM metrics become stable in the 2.0 version, see https://github.com/open-telemetry/semantic-conventions/pull/317

Reply:
Thanks for the detailed explanation @mateuszrzeszutek, I think it makes sense for GC metrics as it may vary depending on the apps and so on. For me, as long as they can be created it's fine, and adding a new View worked perfectly.